tools/DevInsights/DevHome.DevInsights/Strings/en-us/Resources.resw

```diff
@@ -906,12 +906,12 @@
     <comment>Content Dialog button text which on click opens the Advanced App settings page in Windows Settings app.</comment>
   </data>
   <data name="AliasDisabledDialogContent" xml:space="preserve">
-    <value>Go to Windows Settings &gt; Apps &gt; Advanced app settings &gt; App execution alias &gt; Project Ironsides &gt; Enable the toggle and try to launch PI as admin again.</value>
-    <comment>{Locked="Ironsides"} Content Dialog content suggesting users steps to enable app execution alias from Windows Settings app</comment>
+    <value>Go to Windows Settings &gt; Apps &gt; Advanced app settings &gt; App execution alias &gt; Dev Insights &gt; Enable the toggle and try to launch Dev Insights as admin again.</value>
+    <comment>Content Dialog content suggesting users steps to enable app execution alias from Windows Settings app</comment>

```

Comment:
Should Dev Insights be locked here in comments?

Reply:
I'm waiting to heat back to branding.